app/src/main/java/com/nytimes/android/sample/activity/PersistingStoreActivity.java

```diff
@@ -52,9 +54,11 @@ private void initStore() {
         }
     }
 
+    @SuppressWarnings("CheckReturnValue")
```

Comment:
`persistedStore.get.subscribe(...)` return a `Disposable` object, which is unused. This causes [CheckReturnValue](http://errorprone.info/bugpattern/CheckReturnValue) error and the build fails.